bundle-workflow/README.md

```diff
@@ -165,3 +166,26 @@ The following options are available.
 | --component [name] | Test a single component by name, e.g. `--component common-utils`.       |
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
+
+### Auto-Generate Manifests
+
+The [manifests workflow](src/manifests) reacts to version increments in OpenSearch and its components by extracting Gradle properties from project branches. When a new version is identified, it creates a new input manifest in [manifests](../manifests) and [opens a pull request](../.github/workflows/manifests.yml).
```

Comment:
Can we specify which branches of core & components are scanned?

If I'm reading this correctly, we are only scanning branches of core (with the specific 'root' pattern)  and the 'main' branch of component repos.  Is that correct?

Reply:
Correct. We’re looking at anything that seems like a version number. It’s really not that important though, we could be examining all branches.

We should probably not actually be looking at branches like 1.0 to save time - in our branching strategy we always increment on main or 1.x first.